### ISSUE TITLE
Replace outdated docstring in examples using FashionMNIST dataset

### ISSUE THREAD

**Issue by nzw0301** [optuna/optuna-examples#71]
## Motivation

Due to the instability of MNIST download, this repo's example codes have used FashionMNIST instead of MNIST.
However, some docstring is supposed to be mentioning MNIST, `hand-written digit recognition`, even though they use FashionMNIST as follows:

> In this example, we optimize the validation accuracy of hand-written digit recognition using
> Catalyst, and FashionMNIST. We optimize the neural network architecture.

The lines come from https://github.com/optuna/optuna-examples/blob/6a6b20ad634627eebb3e7e104f73b70b45c6e624/pytorch/catalyst_simple.py.

## Description
Revise the docstring for FashionMNIST not MNIST.
